az-digital/az_quickstart
# Closes #1630: Change demo content email addresses to @arizona.edu

## Description
Closes #1630 by changing AZ Person email addresses in the Quickstart Demo Content module from @email.arizona.edu to @arizona.edu.

## Related issues
#1630 (Follow-up issue from #1328)

## How to test


Enable the Quickstart Demo Content module and then check the people directory page or individual Person nodes to confirm that the email addresses end with @arizona.edu.

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
